Security/Core/Authentication/Token/AbstractOAuthToken.php

```diff
@@ -67,7 +67,7 @@ public function __construct($accessToken, array $roles = [])
 
         $this->setRawToken($accessToken);
 
-        // required for compatibility with Symfony 5.4
+        // @deprecated since Symfony 5.4
```

Comment:
Can you revert this? It will be caught most likely by `symfony/phpunit-bridge` but it's not our real deprecation, it's BC layer.

Reply:
done.
I plan to write some tests today